irqbalance.c

```diff
@@ -637,7 +638,8 @@ int main(int argc, char** argv)
 			    "single cpu.  Shutting down\n";
 
 		log(TO_ALL, LOG_WARNING, "%s", msg);
-		exit(EXIT_SUCCESS);
+		ret = EXIT_SUCCESS;
```

Comment:
you don't need this here, ret is already assigned EXIT_SUCCESS in the prologue

Reply:
ok, thanks.